docs/content/any/project/changelogs/2021-06-09.md

```diff
@@ -55,8 +55,6 @@ We have renamed this field to `permissions` for clarity.
 
 Previously, we required role names to be globally unique. Now, role names will be internally namespaced, removing the globally unique requirement. Like permissions, the role namespace is the resource name specified in the `resource` predicate. Roles names must be unique within a single resource namespace. Roles associated with other resources must be referenced using the namespace. Roles within the same resource can be referenced without the namespace.
 
-Below is an example `resource` predicate that reflects the above changes. For a more in-depth example, see our [documentation]({{< ref path="/guides/roles/sqlalchemy" lang="python" >}}) and [example repository](https://github.com/osohq/gitclub/tree/main/backends/flask-sqlalchemy).
```

Comment:
why delete this whole bullet but leave the snippet below, now w/ less context?

Reply:
my bad added back